File: JAVA/6thOct-5.java
import java.io.*;
import java.util.*;

public class Main 
  {
    public static void main(String[] args) 
    {
        Scanner sc=new Scanner(System.in);
        System.out.println("Enter number of terms");
        int n=sc.nextInt();
        
        int n1 = 1, n2 = 1,x;
        
        for (int i = 1; i <= n; i++)
        {
           x=isPrime(n1);
           
           if(n1%5==0||x==1)
             {
               System.out.print(0 + " ");
             }
           else
             {
               System.out.print(n1 + " ");
             }

            int nth = n1 + n2;
            n1 = n2;
            n2 = nth;
        }
    }
    
    public static int isPrime(int n)
    {
      if(n==1)
      {
           return 2;
      }
      else
      {
        int x=1;
        for(int i=2;i<=n/2;i++)
         {
          if(n%i==0)
           {
             x=2;
             break;
           }
         }
       return x;
      }
   }
  }

File: Python/6thOct-5.py

def fibo(num):
l=[1]
a = 0
b = 1
if num <= 0:
print("Enter a non-negative Integer!")
elif num == 1:
print(b) 
else:
for i in range(1,num):
t = a + b
a = b
b = t
if b%5==0 or isPrime(b):
l.append(0)
else:
l.append(b)
print(l) 

def isPrime(no):  
        if no > 1:
           for i in range(2,no):
               if (no % i) == 0:
                   return False
           else:
               return True
        else:
           return False

fibonacci(int(input("Enter Number Of terms: ")))

